test/Microsoft.ML.Tests/TrainerEstimators/SdcaTests.cs

```diff
@@ -130,5 +130,74 @@ public void SdcaSupportVectorMachine()
+
+            // ML.NET doesn't cache data set by default. Caching is very helpful when working with iterative
+            // algorithms which needs many data passes. Since SDCA is the case, we cache.
+            data = mlContext.Data.Cache(data);
+
+            // Step 2: Create a binary classifier.
+            // We set the "Label" column as the label of the dataset, and the "Features" column as the features column.
+
+            var pipeline = mlContext.Transforms.Conversion.MapValueToKey("LabelIndex", "Label").
+                           Append(mlContext.MulticlassClassification.Trainers.Sdca(labelColumnName: "LabelIndex", featureColumnName: "Features", l2Regularization: 0.001f));
+
+            // Step 3: Train the pipeline created.
+            var model = pipeline.Fit(data);
+
+            // Step 4: Make prediction and evaluate its quality (on training set).
+            var prediction = model.Transform(data);
+            var metrics = mlContext.MulticlassClassification.Evaluate(prediction, label: "LabelIndex", topK: 1);
+
+            // Check a few metrics to make sure the trained model is ok.
+            Assert.InRange(metrics.TopKAccuracy, 0.8, 1);
+            Assert.InRange(metrics.LogLoss, 0, 0.5);
+        }
+
+        [Fact]
+        public void SdcaMulticlassSupportVectorMachine()
+        {
+            // Generate C# objects as training examples.
+            var rawData = SamplesUtils.DatasetUtils.GenerateFloatLabelFloatFeatureVectorSamples(512);
+
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Step 1: Read the data as an IDataView.
+            var data = mlContext.Data.LoadFromEnumerable(rawData);
+
+            // ML.NET doesn't cache data set by default. Caching is very helpful when working with iterative
+            // algorithms which needs many data passes. Since SDCA is the case, we cache.
+            data = mlContext.Data.Cache(data);
+
+            // Step 2: Create a binary classifier.
+            // We set the "Label" column as the label of the dataset, and the "Features" column as the features column.
+            var pipeline = mlContext.Transforms.Conversion.MapValueToKey("LabelIndex", "Label").
+                Append(mlContext.MulticlassClassification.Trainers.SdcaNonCalibrated(labelColumnName: "LabelIndex", featureColumnName: "Features", loss: new HingeLoss(), l2Regularization: 0.001f));
+
+            // Step 3: Train the pipeline created.
+            var model = pipeline.Fit(data);
+
+            // Step 4: Make prediction and evaluate its quality (on training set).
+            var prediction = model.Transform(data);
+            var metrics = mlContext.MulticlassClassification.Evaluate(prediction, label: "LabelIndex", topK: 1);
```

Comment:
>label [](start = 82, length = 5)

I think @artidoro rename them to labelColumnName in his latest PR.

Reply:
Merged with master. Thanks.

---
In reply to: [266997235](https://github.com/dotnet/machinelearning/pull/2976#discussion_r266997235) [](ancestors = 266997235)